app/views/layouts/_lte_sidebar.html.erb

```diff
@@ -101,6 +101,11 @@
           </li>
         </ul>
       </li>
+      <li class="<%= 'active' if current_page?(organization_path) %>">
+        <%= link_to(organization_path) do %>
+          <i class="fa fa-circle-o"></i> <span>Organizations</span>
```

Comment:
How about using `fa-home` and calling it `My Organization`

Reply:
sounds great